src/pages/AnimeOnsen/meta.json

```diff
@@ -1,6 +1,6 @@
 {
-  "search": "https://animeonsen.xyz/search/{searchterm}",
+  "search": "https://www.animeonsen.xyz/search/{searchterm}",
   "urls": {
-    "match": ["*://animeonsen.xyz/*"]
+    "match": ["*://animeonsen.xyz/*", "*://www.animeonsen.xyz/*"]
```

Comment:
`"*://*.animeonsen.xyz/*"` should match both

Reply:
> `"*://*.animeonsen.xyz/*"` should match both

The site is strictly moving to using the `www.` subdomain, other subdomains are reserved for other functions that may not support MALSync. Weird things might happen if using the wildcard.